traits/core.md

```diff
@@ -0,0 +1,87 @@
+# Core traits
+
+## Manual Scaler
+
+This trait definition allows operators to manually scale the number of replicas for workload types that allow scaling operations.
+
+### Applicable Workloads
+
+ - `core.oam.dev/v1alpha2.containerWorkload`
```

Comment:
Can we agree on `ContainerisedWorkload` (or `ContainerizedWorkload` if we prefer en-us 😉)? In particular I'm pretty sure the lowercase `c` in `container` makes it an invalid CR kind.

Reply:
This `group/version.kind` representation of GVK still feels a little unusual to me - like we're using Kubernetes concepts (GVK) but inventing an OAM specific way of representing them. I wonder whether we should just stick to API convention and use separate fields. i.e.:

```yaml
spec:
  appliesTo:
  - apiVersion: core.oam.dev/v1alpha2
    kind: ContainerisedWorkload
```

One interesting side effect of this approach is that we could make `kind` optional, allowing a trait to apply to all core workload kinds by writing:

```yaml
spec:
  appliesTo:
  - apiversion: core.oam.dev/v1alpha2
```

Reply:
yes, we can use ContainerizedWorkload (we are in US 🥇 ).

Reply:
Maybe we could discuss about this way:

```
spec:
  appliesTo:
  - apiVersion: core.oam.dev/v1alpha2
    kind: ContainerizedWorkload
```

Currently, I'll change to 

```
  appliesToWorkloads:
    - core.oam.dev/v1alpha2.ContainerizedWorkload
```